libs/perun/components/src/lib/password-reset/password-reset.component.ts

```diff
@@ -69,11 +64,39 @@ export class PasswordResetComponent implements OnInit {
     });
   }
 
-  resetPassword(login: string): void {
-    window.open(
-      this.otherApplicationService.getUrlForOtherApplication(AppType.PwdReset, login),
-      '_blank'
-    );
+  resetPassword(login: Attribute): void {
+    if (!this.authenticationPage) {
+      void this.router.navigate([], {
+        queryParams: {
+          namespace: login.friendlyNameParameter,
+        },
+        queryParamsHandling: 'merge',
+      });
+    }
```

Comment:
I believe the added query parameter is not used for anything, am I missing something ? If it is in fact not needed please remove the navigation here and after dialog is closed section.